src/main/java/ddingdong/ddingdongBE/domain/club/api/CentralClubApi.java

```diff
@@ -0,0 +1,125 @@
+package ddingdong.ddingdongBE.domain.club.api;
+
+import ddingdong.ddingdongBE.auth.PrincipalDetails;
+import ddingdong.ddingdongBE.common.exception.ErrorResponse;
+import ddingdong.ddingdongBE.domain.club.controller.dto.request.UpdateClubMemberRequest;
+import ddingdong.ddingdongBE.domain.club.controller.dto.request.UpdateClubRequest;
+import ddingdong.ddingdongBE.domain.club.controller.dto.response.DetailClubResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import javax.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.multipart.MultipartFile;
+
+@Tag(name = "Club - Club", description = "Club CentralClub API")
+@RequestMapping("/server/club/my")
+public interface CentralClubApi {
+
+    @Operation(summary = "동아리원 명단 다운로드 API")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "명단 다운로드 성공")})
+    @ResponseStatus(HttpStatus.OK)
+    @SecurityRequirement(name = "AccessToken")
+    @GetMapping("/club-members/excel")
+    ResponseEntity<byte[]> getMyClubMemberListFile(@AuthenticationPrincipal PrincipalDetails principalDetails);
+
+    @Operation(summary = "내 동아리 정보 조회 API")
+    @ResponseStatus(HttpStatus.OK)
+    @SecurityRequirement(name = "AccessToken")
+    @GetMapping
+    DetailClubResponse getMyClub(@AuthenticationPrincipal PrincipalDetails principalDetails);
+
+    @Operation(summary = "내 동아리 정보 수정 API")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @SecurityRequirement(name = "AccessToken")
+    @PatchMapping()
```

Comment:
```suggestion
    @PatchMapping
```
p3) () 없애도 좋을 것 같습니다!